### PR TITLE
Add yum_install_allows_manual_input query Closes #1088

### LINKED ISSUE #1088: Add y_switch_avoid_manual_input query for Docker
**Platform**
Docker

**Provider**
Dockerfile

**Description**
Check if we are using the -y switch to avoid manual input

### PR DESCRIPTION
**Pull request by NunoAraujoCX**
Closes #1088 
Check if we are using -y to avoid manual input 'yum install -y <package>'

### CODE REVIEW

File: assets/queries/dockerfile/y_switch_avoid_manual_input/query.rego [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].command[name][_]
+  contains(resource.Value[j], "yum install")
```
Comment (joaoReigota1) [inline assets/queries/dockerfile/y_switch_avoid_manual_input/query.rego:5]: Need to check for "yum groupinstall", "yum localinstall" as well

File: assets/queries/dockerfile/y_switch_avoid_manual_input/query.rego [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  not contains(resource.Value[j], "-y")
```
Comment (joaoReigota1) [inline assets/queries/dockerfile/y_switch_avoid_manual_input/query.rego:6]: Need to check for flag "--assumeyes" as well

File: assets/queries/dockerfile/y_switch_avoid_manual_input/metadata.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "y_switch_avoid_manual_input",
+  "queryName": "Y Switch Avoid Manual Input",
```
Comment (joaoReigota1) [inline assets/queries/dockerfile/y_switch_avoid_manual_input/metadata.json:3]: The query name should reflect the vulnerability in an easy and comprehensible way
"yum_install_allows_manual_input"

**Review by joaoReigota1 — CHANGES_REQUESTED** [review]

**Review by joaoReigota1 — COMMENTED** [review]
Approved

**Review by joelsou5a — APPROVED** [review]

**Pull request merged**